app/assets/javascripts/registration.js.coffee

```diff
@@ -0,0 +1,15 @@
+$ ->
+  $('#registration-button').on 'click', () ->
+    if $('#new_location').length == 0
+      $.get I18n.locale + '/locations/new', (data) ->
+        $('#registration').html data
+
+  $('#sidebar-left').on('ajax:success', '#new_location', (e, data, status, xhr) ->
+    $('#registration').html xhr.responseText
+  ).on('ajax:error', '#new_location', (e, xhr, status, error) ->
+    $('#new_location').prepend "<div class='alert alert-danger'>Server error.</div>"
```

Comment:
True, but that would require js-i18n. For now I'm thinking how to bypass that problem and to be able to throw that library away (to save some file transfer size).